README.md

```diff
@@ -1,3 +1,71 @@
 # Hey Rodney dataset
 
 This is a dataset of voice samples for our wake word detection of "Hey Rodney".
+
+## Steps to Record audio data and upload it on github
+
+## Specifications:
+
+| __Phrase__ | "Hey Rhodney"   |
+| ------------- | ------------- |
+| __Number of Samples__  | 3  |
+| __Sample Rate__ | 16 KHz  |
+| __Number of Channels__ | 1 |
+| __Encoding__ | Signed 16-bit PCM |
+
+
+
+## Step 1: Record the Audio
+
+### Steps to implement in audacity:
+1. Open audacity 
+2. Set Project rate to 16000 ( In the bottom left ) 
+3. Set the recording channel 1 (mono) recording channel ( which is in the bar just above the timestamps )
+4. Record your voice saying *“Hey Rodney”*
+
+
+__Export the audio file in the form of a .wav file__:<br> 
+*Create a dedicated folder to store all recordings*
+1. After completing the recording 
+2. Go to File
+3. Export
+4. Export as *.wav*
+
+## Step 2: Create a Zip
+
+
+### Ways to zip a file: 
+
+__Mac__: 
+- Right click on the folder 
+- Compress
+
+
+__Windows__: 
+- Right click on the folder
+- Send to  
+- Compressed (zipped) 
+- Folder
+
+__Linux__: 
+
+```zip -r [Name of the new folder.zip] [Name of the folder that contains the voice samples]```
+
+__Using a website__:  
+- Go to the website  (https://www.ezyzip.com/zip-folder-online.html#) 
+choose the desired folder to be zipped 
+- Upload the files 
+- Click “Zip Files” 
+- Click “save zip file”
+
+
+## Step 3: Upload the data on GitHub
+
+### Upload the folder to the GitHub repository *“hey-rodney-dataset”* by creating an issue:
+
+- Open the url : https://github.com/rishabh-vasudevan/hey-rodney-dataset
+- Go to the issues tab on the top  
```

Comment:
I think you could directly link to the issues page. Maybe even to the "New Issue" page.

Reply:
yes that actually will be better